server/src/telemetry/SentryServerTelemetry.ts

```diff
@@ -47,9 +47,17 @@ export class SentryServerTelemetry implements Telemetry {
       throw new Error("Tracing not loaded");
     }
 
+    const traceSampleRate = this.serverState.env === "development" ? 1 : 0.001;
+
     Sentry.init({
       dsn: this.dsn,
-      tracesSampleRate: serverState.env === "development" ? 1 : 0.001,
+      tracesSampler: ({ transactionContext: { name } }) => {
```

Comment:
this means we will no longer have sampling for both the user platform AND the 2 features flag status (as they are recorded within `indexing`). Is that correct?

Reply:
Yes. The `indexing` event won't be sampled, as the feature flag status is recorded into this event, neither will the feature flag tags.
Put another way sampling will no longer obscure the `indexing` based recording of feature flags.